### ISSUE TITLE
Support Tokio runtime

### ISSUE THREAD

**Issue by jsdw** [async-email/async-smtp#13]
Hello!

Would you consider adding support or accepting a PR that adds support for tokio in addition to async-std?

I would envisage using feature flags in the same way that something like https://github.com/launchbadge/sqlx does.

**Comment by dignifiedquire** [comment]
yes, I think it would be great being able to switch based on a flag

**Comment by manuelpelloni** [comment]
Lettre is adding tokio support in https://github.com/lettre/lettre/pull/449, and it looks like they are planning to release it soon in their 0.10.0-alpha.2 release
